docs/src/pages/components/text-fields/CustomizedInputs.tsx

```diff
@@ -80,13 +80,13 @@ const useStylesReddit = makeStyles((theme: Theme) =>
       border: '1px solid #e2e2e1',
       overflow: 'hidden',
       borderRadius: 4,
-      backgroundColor: '#fcfcfb',
+      backgroundColor: theme.palette.background.paper,
       transition: theme.transitions.create(['border-color', 'box-shadow']),
       '&:hover': {
-        backgroundColor: '#fff',
+        backgroundColor: theme.palette.action.hover,
```

Comment:
Would it work better?

```suggestion
        backgroundColor: 'transparent',
```